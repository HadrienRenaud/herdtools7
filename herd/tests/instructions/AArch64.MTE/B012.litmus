AArch64 B012
{
int x[8]={1,2,3,4,5,6,7,8};

0:X0=x:green; 0:X1=x:red;
}
P0                ;
STG X1,[X0,#16]   ;
DSB SY            ;
MOV W2,#9         ;
STR W2,[X1,#16]    ;

forall ~fault(P0,x)
