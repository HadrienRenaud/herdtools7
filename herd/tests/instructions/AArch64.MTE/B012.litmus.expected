Test B012 Required
States 1
  ~Fault(P0,x);
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (not (fault(P0,x)))
Observation B012 Always 1 0
Hash=373fea60dc2524676ae0d6095296613a

